reqs/views.py

```diff
@@ -141,6 +143,12 @@ def filter_queryset(self, request, queryset, view):
             return self.priority_ordering(request, queryset)
 
 
+class HiddenPolicyFilter(BaseFilterBackend):
+
+    def filter_queryset(self, request, queryset, view):
+        return queryset.exclude(policy__nonpublic=True)
```

Comment:
As I understand it, these `Filter` classes are for dynamic (per request) logic. Should we just override `get_queryset` instead?